src/client/src/components/LeftSidebar/index.tsx

```diff
@@ -22,25 +18,21 @@ const messages = defineMessages({
     id: "leftSidebar.newProject",
     defaultMessage: "1. New Project"
   },
-  projectType: {
-    id: "leftSidebar.projectType",
-    defaultMessage: "2. Project Type"
-  },
   frameworks: {
     id: "leftSidebar.frameworks",
-    defaultMessage: "3. Frameworks"
+    defaultMessage: "2. Frameworks"
```

Comment:
Would it make more sense to have these numbers be auto generated based on page order? 

```javascript
`${index}. ${defaultMessage}` 
```

Reply:
Yea it would make more sense - oo I did not know numbers need to be i18n'd

Reply:
I think most notably right-to-left languages are going to be different. But also, other languages have different [number symbols](https://en.wikipedia.org/wiki/Hindu%E2%80%93Arabic_numeral_system#Symbols) too, even if 0-9 symbols are understood in a modern context in a given locality.

Reply:
Yeah, ``${index}. ${defaultMessage}` ` wouldn't work for rtl languages, so maybe it's better to leave as-is and localize the whole thing?

Reply:
I will leave this as is